### PR TITLE
Release preparations

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -4,3 +4,6 @@
 [submodule "pression/data/snappy"]
 	path = pression/data/snappy
 	url = https://github.com/google/snappy
+[submodule "CMake/common"]
+	path = CMake/common
+	url = https://github.com/Eyescale/CMake
```
Comment (tribal-tec) [inline .gitmodules]: no shallow?

**Comment by eile** [comment]
> no new lunchbox sha-1?
I did a Pression-update-git-subprojects, verifying...

**Comment by eile** [comment]
both done.

**Comment by tribal-tec** [comment]
maybe wait for https://github.com/Eyescale/Lunchbox/pull/311?

**Pull request merged**